README.md

```diff
@@ -1,16 +1,16 @@
 # Web-ext
 
 This is a command line tool to help build, run, and test
-[web extensions](https://wiki.mozilla.org/WebExtensions).
+[WebExtensions](https://wiki.mozilla.org/WebExtensions).
 
 [![Build Status](https://travis-ci.org/mozilla/web-ext.svg?branch=master)](https://travis-ci.org/mozilla/web-ext)
 [![Coverage Status](https://coveralls.io/repos/github/mozilla/web-ext/badge.svg?branch=master)](https://coveralls.io/github/mozilla/web-ext?branch=master)
 [![Dependency Status](https://david-dm.org/mozilla/web-ext.svg)](https://david-dm.org/mozilla/web-ext)
 [![devDependency Status](https://david-dm.org/mozilla/web-ext/dev-status.svg)](https://david-dm.org/mozilla/web-ext#info=devDependencies)
 
-Ultimately, it aims to support web extensions in a standard, portable,
+Ultimately, it aims to support WebExtensions in a standard, portable,
```

Comment:
For this case, I think it would be better to say "it aims to support browser extensions in a standard..."

See https://wiki.mozilla.org/Add-ons/Terminology#Glossary